pkg/internal/resources/catapult/catapult.go

```diff
@@ -233,6 +234,7 @@ func Manifests(c Config) ([]unstructured.Unstructured, error) {
 						},
 					},
 				},
+				SideEffects: &sideEffectsDryRun,
```

Comment:
Wait....this is PR regarding elevator, why are you touching catapult code?

Reply:
Also which exactly side-effect are we talking about here that you've chosen `NoneOnDryRun` ?

Reply:
This is because the `SideEffects` is `Unknown` by default, and requests with the dryRun attribute will be auto-rejected by the webhook, since we need to make dry-run reqests in the elevator webhook to fight against with the catapult webhook, so we need to specify the `SideEffect` for it.
https://github.com/kubernetes/kubernetes/blob/1c974109b672ade9aae86f2daf1d50c89c36949f/staging/src/k8s.io/api/admissionregistration/v1beta1/types.go#L282